<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Member</class>
 <widget class="QDialog" name="Member">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="l_dialogTitle">
     <property name="text">
      <string>Mitglied hinzufügen</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="rightMargin">
      <number>100</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Mitglied</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cb_member">
       <property name="placeholderText">
        <string>Mitglied...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Vorname</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_first_name"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nachname</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_surname"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Matrikelnummer</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="le_student_number"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>E-Mail GLR</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="le_email_glr"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>E-Mail Privat</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="le_email_private"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Telefon</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="le_phone"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Anschrift</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="le_address"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Hemdgröße</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="le_shirt_size"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>VDI-Nummer</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="le_vdi_number"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Sprache</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="le_language"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Verlauf</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tw_changes"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="tb_add">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/img/icon_add.png</normaloff>:/img/icon_add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tb_remove">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/img/icon_remove.png</normaloff>:/img/icon_remove.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Seit</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="le_since"/>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Bis</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLineEdit" name="le_until"/>
     </item>
     <item row="14" column="1">
      <widget class="QLineEdit" name="le_position"/>
     </item>
     <item row="15" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pb_okay">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Okay</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Position</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_member</tabstop>
  <tabstop>le_first_name</tabstop>
  <tabstop>le_surname</tabstop>
  <tabstop>le_student_number</tabstop>
  <tabstop>le_email_glr</tabstop>
  <tabstop>le_email_private</tabstop>
  <tabstop>le_phone</tabstop>
  <tabstop>le_address</tabstop>
  <tabstop>le_shirt_size</tabstop>
  <tabstop>le_vdi_number</tabstop>
  <tabstop>le_language</tabstop>
  <tabstop>tw_changes</tabstop>
  <tabstop>tb_add</tabstop>
  <tabstop>tb_remove</tabstop>
  <tabstop>le_since</tabstop>
  <tabstop>le_until</tabstop>
  <tabstop>le_position</tabstop>
  <tabstop>pb_okay</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
